src/crypto/crypto.gni

```diff
@@ -16,6 +16,8 @@ declare_args() {
   # Crypto implementation: mbedtls, openssl, boringssl, platform.
   chip_crypto = ""
   chip_with_se05x = 0
+  chip_enable_trustm = false
+  chip_enable_trustm_da = false
```

Comment:
Please document these individually, and add `infineon` such as `chip_enable_infineon_trustm`

Reply:
Hi @tcarmelveilleux, Thanks for your review. We have fixed this in the latest commit.